src/Core/Scheduler/Scheduler.js

```diff
@@ -160,7 +160,7 @@ Scheduler.prototype.execute = function execute(command) {
 
     // parse host
     const layer = command.layer;
-    const host = layer.url ? new URL(layer.url, document.location).host : undefined;
+    const host = layer.source && layer.source.url ? new URL(layer.source.url, document.location).host : undefined;
```

Comment:
it doesn't work for 3Dtiles by examples?

Reply:
I see no problem in the 3dtiles example, is this what you meant ?